packages/preview-service/server/routes/api.ts

```diff
@@ -0,0 +1,58 @@
+import zlib from 'zlib'
+import express from 'express'
+import { getObjectsStreamFactory } from '../../repositories/objects'
+import { SpeckleObjectsStream } from '../../utils/speckleObjectsStream'
+import { pipeline, PassThrough } from 'stream'
+import type { Knex } from 'knex'
+
+const apiRouterFactory = (deps: { db: Knex }) => {
+  const { db } = deps
+  const apiRouter = express.Router()
+
+  const isSimpleTextRequested = (req: express.Request) =>
```

Comment:
This could be a helper imported from a helper module, since I saw other endpoints also using a check like this